src/desktop_notifier/base.py

```diff
@@ -115,10 +117,12 @@ class Notification:
         callback will be called without any arguments.
     :param on_dismissed: Callback to call when the notification is dismissed. The
         callback will be called without any arguments.
-    :attachment: URI for an attachment to the notification.
-    :param sound: Whether to play a sound when the notification is shown.
+    :param attachment: URI for an attachment to the notification.
+    :param sound: [DEPRECATED] Use sound_file=DEFAULT_SOUND instead.
     :param thread: An identifier to group related notifications together.
     :param timeout: Duration for which the notification in shown.
+    :param sound_file: String identifying the sound to play when the notification is
```

Comment:
nit: How about `sound_name` instead? This offers the option to add an explicit `sound_file` argument in the future to pass a file path instead of a name.

Reply:
tbhi just went with `sound_file` bc you suggested it. that said i don't think it's the worst option? it's maybe nice that `sound_file` is a bit nebulous as IMHO it makes more sense to just take a single string arg (or maybe `str | Path`) than to have two separate args `sound_name` and `sound_file_path` or whatever that can't be used at the same time.

that said if one were to take the long term `str | Path` route there still might an argument name that's better than `sound_file`.

just my $0.02; it's your package so i'm happy to do whatever you prefer.

Reply:
SGTM, let's keep it as is.